drizzlepac/haputils/photometry_graphics.py

```diff
@@ -63,21 +75,20 @@ def compute_global_stats(value_array):
     return mean_value
 
 
-def load_data_to_arrays(csv_filename):
-    """Load the harvested data, stored in a CSV file, into local arrays.
+def get_data(storage_filename):
+    """Load the harvested data, stored in a storage file, into local arrays.
 
     Parameters
     ==========
-    csv_filename: str
-    Name of the CSV file created by generate_photometry_df.
+    storage_filename: str
```

Comment:
```suggestion
    storage_filename : str
```
Just a small adjustment to allow Sphinx to properly format the docstring.